app/controlplane/internal/service/attestation.go

```diff
@@ -213,9 +213,17 @@ func (s *AttestationService) Store(ctx context.Context, req *cpAPI.AttestationSe
 		return nil, errors.NotFound("not found", "robot account not found")
 	}
 
-	// Check that provided workflowRun belongs to workflow encoded in the robot account
-	if exists, err := s.wrUseCase.ExistsInWorkflow(ctx, robotAccount.WorkflowID, req.WorkflowRunId); err != nil || !exists {
-		return nil, errors.NotFound("not found", "workflowRun not found")
+	if robotAccount.WorkflowID == "" {
```

Comment:
I thought that we said that we were ok with just checking if it belongs to an org, in both cases.